src/mango/test/19-find-conflicts.py

```diff
@@ -12,12 +12,13 @@
 
 import mango
 import copy
+import unittest
 
 DOC = [{"_id": "doc", "a": 2}]
 
 CONFLICT = [{"_id": "doc", "_rev": "1-23202479633c2b380f79507a776743d5", "a": 1}]
 
-
+@unittest.skip("re-enable once conflicts are supported")
```

Comment:
The couch_views_indexer code loads conflicts so this shouldn't be disabled, correct?

Reply:
I've only just merged that code in to the prototype/fdb-layer today so I can make that change